nvme-print-stdout.c

```diff
@@ -239,7 +239,7 @@ static void stdout_add_bitmap(int i, __u8 seb)
 {
 	for (int bit = 0; bit < CHAR_BIT; bit++) {
 		if (nvme_pel_event_to_string(bit + i * CHAR_BIT)) {
-			if ((seb >> bit) & 0x1)
+			if (BIT_CHECK(seb, bit))
```

Comment:
Let's leave as it is. The macros always hide what is happening and it means when you don't know what it does you need to look it up. The original code is not that bad :)

Reply:
Okay understood then delete the commit to use BIT_CHECK() macro and rebased the branch with the latest master. Thank you. (But still I am not sure what is good cases to use macros and bad cases.. Since already nvme-cli using macros NVME_GET() and NVME_SET(), etc. in libnvme and also in linux kernel there are macros BIT(), BIT_MASK() and GEN_MASK(), etc. so if those macros are not good seems needed to be deleted to unify.)

Reply:
I hear you and yes, it's a bit of muddy water. My personal taste is to avoid macros if possible due to the mentioned arguments above. Adding indirection/abstraction can be hard to maintain. In the end it's also personal preference here.

Though when the the code gets difficult to follow or we want to keep all the definition in one place, it makes sense to have macros like  NVME_GET/NVME_SET around. 

For simple bit operations I think the C API is simple enough to read. 

BTW, the BIT_CHECK macro in the kernel has fans and haters in the kernel. So not all maintainers want to see them in their subsystem :)

Reply:
Thank you so much for your explanation. Now I could understand much more before.